macros/utils/data_types/data_size.sql

```diff
@@ -1,15 +1,23 @@
-{% macro get_column_size() %}
-  {{ return(adapter.dispatch('get_column_size', 'elementary')()) }}
+{% macro long_string_size() %}
```

Comment:
Considering deleting this macro altogether (not used anymore) to discourage design in which this matters.
What do you think? @haritamar